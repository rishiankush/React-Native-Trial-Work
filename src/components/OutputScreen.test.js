import React from 'react';
import renderer from 'react-test-renderer';
import OutputScreen from './OuputScreen';

test('renders correctly', () => {
  const tree = renderer.create(<OutputScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});