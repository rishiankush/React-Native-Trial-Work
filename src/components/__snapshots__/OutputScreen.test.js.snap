// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#000000",
          "fontFamily": "System",
          "fontWeight": "400",
        },
        Object {
          "textAlign": "left",
        },
        Array [
          Object {
            "textAlign": "left",
          },
          Object {
            "color": "rgba(0, 0, 0, 0.54)",
            "fontFamily": "System",
            "fontWeight": "400",
            "writingDirection": "ltr",
          },
          Array [
            Object {
              "fontSize": 12,
              "letterSpacing": 0.4,
              "lineHeight": 20,
              "marginVertical": 2,
            },
            Object {
              "marginTop": "20%",
            },
          ],
        ],
      ]
    }
  >
    Showing next business date from the input date.
  </Text>
  <Text
    style={
      Array [
        Object {
          "color": "#000000",
          "fontFamily": "System",
          "fontWeight": "400",
        },
        Object {
          "textAlign": "left",
        },
        Array [
          Object {
            "textAlign": "left",
          },
          Object {
            "color": "rgba(0, 0, 0, 0.87)",
            "fontFamily": "System",
            "fontWeight": "400",
            "writingDirection": "ltr",
          },
          Array [
            Object {
              "fontSize": 24,
              "letterSpacing": 0,
              "lineHeight": 32,
              "marginVertical": 2,
            },
            undefined,
          ],
        ],
      ]
    }
  >
    May 7th 2021
  </Text>
</View>
`;
